src/components/bookmarks/bookmarkDropdown/bookmarkDropdown.js

```diff
@@ -191,6 +219,7 @@ class Bookmark extends React.Component {
               name="accountName"
               value={fields.accountName.value}
               placeholder={t('ie. Lisker123')}
+              onClick={this.handleInputClick}
               readOnly={fields.accountName.isReadOnly}
               className={`${styles.input} ${fields.accountName.error ? 'error' : ''}`}
               feedback={accountName.feedback}
```

Comment:
The input should have
```
                size="xs"
```
Based on the design.

Reply:
✅